pulseapi/profiles/serializers.py

```diff
@@ -100,6 +100,19 @@ class UserProfileSerializer(serializers.ModelSerializer):
     program_type = serializers.StringRelatedField()
     program_year = serializers.StringRelatedField()
 
+    entry_count = serializers.SerializerMethodField()
+
+    def get_entry_count(self, obj):
+        entry_queryset = Entry.objects.public()
+
+        return {
+            'created': obj.related_entry_creators.filter(entry__in=entry_queryset).count(),
+            'published': entry_queryset.filter(
+                published_by=obj.related_user
+            ).count() if getattr(obj, 'related_user', None) else 0,
```

Comment:
Just for logic's sake, `if getattr(obj, 'related_user', False) else 0` probably makes more sense?